packages/block-editor/src/components/block-toolbar/utils.js

```diff
@@ -195,14 +195,15 @@ export function useToggleBlockHighlight( clientId ) {
 		if ( requestAnimationFrameId ) {
 			cancelAnimationFrame( requestAnimationFrameId );
 		}
+
 		return () => {
 			// Sequences state change to enable editor updates (e.g. cursor
 			// position) to render correctly.
 			requestAnimationFrameId = requestAnimationFrame( () => {
```

Comment:
Where is this cancelled?

Reply:
@ellatrix I don't think we need the RAF. I paused on this for now, as regardless of whether I use RAF or not, a couple of particular E2E tests keep failing. They fail when they attempt to use the `toggleBlockHighlight` action in a `useCallback` return.